pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/FrontendAnalyticsRoute.java

```diff
@@ -1,37 +0,0 @@
-package org.broadinstitute.dsm.route;
-
-import org.broadinstitute.dsm.analytics.GoogleAnalyticsMetrics;
-import org.broadinstitute.dsm.analytics.GoogleAnalyticsMetricsTracker;
-import org.broadinstitute.dsm.security.RequestHandler;
-import org.broadinstitute.dsm.statics.RoutePath;
-import org.broadinstitute.lddp.handlers.util.Result;
-import spark.QueryParamsMap;
-import spark.Request;
-import spark.Response;
-
```

Comment:
I couldn't find any use of this route in frontend code, and since the route just posts some timing information to GA, I decided to delete it entirely.  CC @pegahtah